docs/drvfc.rst

```diff
@@ -0,0 +1,172 @@
+=======================
+Firecracker driver
+=======================
+
+.. contents::
+
+`Firecracker <https://github.com/firecracker-microvm/firecracker>`__ is an open source Virtual Machine Monitor (VMM) that
+runs on top of KVM. The focus of the project is on running VM's in a
+light-weight, secure way providing multi-tenant support. The features
+supported are minimalist to ensure the performance and security standards.
+
+The libvirt Firecrakcer driver is intended to be run as a system
+driver. The firecracker binary itself should be
+available in the path.
+
+Expected connection URI would be
+
+``fc:///system``
+
+
+Example guest domain XML configurations
+=======================================
+
+The following docs were written using Firecracker version ``v0.26``
```

Comment:
isn't it v0.25?

We should state the versions this driver works on (min v0.25.0 and lower than v1.0)

Reply:
It works with both AFAIK, I will update this.